.browserslistrc

```diff
@@ -1,3 +1,3 @@
 last 2 versions
```

Comment:
add to `last 2 versions and not dead`

and would go with `> 0` i have strange feeling:

```
last 2 versions and not dead
not ie < 12
not ie_mob < 12
```

this will throw away exotic not longer used browsers here the result differnent of `npx browserslist`:

![Bildschirmfoto 2019-08-29 um 09 45 15](https://user-images.githubusercontent.com/1698337/63920733-c2b77a00-ca41-11e9-8362-5a9793821acc.png)